TOC.md

```diff
@@ -35,10 +35,6 @@ product and design decisions.
   - Establish and maintain the overall technical governance guidelines for the
     project.
 
-  - Decide which sub-projects are part of the Cloud Foundry project, including
-    accepting new sub-projects and pruning existing sub-projects to maintain
-    community focus
-
```

Comment:
I am all for removing undefined and duplicative terms. The only question I would raise is if the whole thing belongs into the technical or community governance section.
The former already has (and had in addition of what you propose to remove) the following:
>Approve the creation and dissolution of working groups and approve leadership changes of working groups.